src/lib/firebase.ts

```diff
@@ -75,8 +75,14 @@ export class Firebase {
   // Paths
   //
 
+  public getClassPath(user: UserModelType) {
+    return `classes/${user.classHash}`;
+  }
+
   public getUserPath(user: UserModelType, userId?: string) {
-    return `users/${userId || user.id}`;
+    return urlParams.testMigration
```

Comment:
@kswenson I realized 2 things during testing: 1) it is a pain to remember the `testMigration=true` url parameter and 2) I never want to run this branch with `testMigration=false`.

I think what this means is that rather than checking the `testMigration` url parameter here, we could just write:

```
    return FIREBASE_ROOT_OVERRIDE
                 ? `${this.getClassPath(user)}/users/${userId || user.id}`
                 : `users/${userId || user.id}`;
```

This remains safe to merge and easy to update post-migration. What do you think?

Reply:
I've actually reversed my opinion on this - I realized there *is* a situation where I want to run this with `testMigration=false`. When we're testing this branch, it's helpful to switch back to "production mode" to make sure that master will still work when the branch is merged.

I did realize it should say `urlParams.testMigration === "true"` though, since the param is a string instead of a boolean...